addons/binding/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/handler/HDPowerViewHubHandler.java

```diff
@@ -0,0 +1,189 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+/**
+e * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.hdpowerview.handler;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.binding.BaseBridgeHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.openhab.binding.hdpowerview.HDPowerViewBindingConstants;
+import org.openhab.binding.hdpowerview.config.HDPowerViewHubConfiguration;
+import org.openhab.binding.hdpowerview.config.HDPowerViewSceneConfiguration;
+import org.openhab.binding.hdpowerview.config.HDPowerViewShadeConfiguration;
+import org.openhab.binding.hdpowerview.internal.HDPowerViewWebTargets;
+import org.openhab.binding.hdpowerview.internal.api.responses.Scenes;
+import org.openhab.binding.hdpowerview.internal.api.responses.Scenes.Scene;
+import org.openhab.binding.hdpowerview.internal.api.responses.Shades;
+import org.openhab.binding.hdpowerview.internal.api.responses.Shades.Shade;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link HDPowerViewHubHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Andy Lintner - Initial contribution
+ */
+public class HDPowerViewHubHandler extends BaseBridgeHandler {
+
+    private Logger logger = LoggerFactory.getLogger(HDPowerViewHubHandler.class);
+
+    private long refreshInterval;
+
+    private final Client client = ClientBuilder.newClient();
+    private HDPowerViewWebTargets webTargets;
+    private ScheduledFuture<?> pollFuture;
+
+    public HDPowerViewHubHandler(Bridge bridge) {
+        super(bridge);
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+
+    }
+
+    @Override
+    public void initialize() {
+        HDPowerViewHubConfiguration config = getConfigAs(HDPowerViewHubConfiguration.class);
```

Comment:
You can call `BaseThingHandler.validateConfigurationParameters` to have some basic validation done, but this currently does not check against format pattern.
@tomhoefer: It would make sense to add such a check, would you agree? Btw, when calling `validateConfigurationParameters` within `initialize()`, should we allow the method to throw a `ConfigValidationException` as well (currently you have to catch it in the code yourself)?